src/test/java/Homework18.java

```diff
@@ -0,0 +1,52 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+
+
+    @Test
+    public void playSong() throws InterruptedException {
+
+
+        //Navigate to "https://qa.koel.app/".
+        navigateToPage();
+        //Log in
+        enterEmail("christina.taylor@testpro.io");
+        enterPassword("jKV0uSX6z1dv");
+        submit();
+        Thread.sleep(1000);
+        //Click «Play next song» (media player controls),
+        clickPlayBtn();
+        Thread.sleep(1000);
+        // then the Play button, to play a song.
+        clickPlayNext();
+        Thread.sleep(1000);
+        // or the pause button is displayed.
+        clickPauseBtn();
+        Thread.sleep(1000);
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![18](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/75b62872-61aa-4bac-9895-2757e3c4cdd5)